addons/advanced_ballistics/functions/fnc_handleFired.sqf

```diff
@@ -59,22 +56,24 @@ _AmmoCacheEntry params ["_airFriction", "_caliber", "_bulletLength", "_bulletMas
 _WeaponCacheEntry params ["_barrelTwist", "_twistDirection", "_barrelLength"];
 
 
-_bulletVelocity = velocity _projectile;
-_muzzleVelocity = vectorMagnitude _bulletVelocity;
+private _bulletVelocity = velocity _projectile;
+private _muzzleVelocity = vectorMagnitude _bulletVelocity;
 
-_barrelVelocityShift = 0;
+private _barrelVelocityShift = 0;
 if (GVAR(barrelLengthInfluenceEnabled)) then {
     _barrelVelocityShift = [_barrelLength, _muzzleVelocityTable, _barrelLengthTable, _muzzleVelocity] call FUNC(calculateBarrelLengthVelocityShift);
 };
 
-_ammoTemperatureVelocityShift = 0;
+private _ammoTemperatureVelocityShift = 0;
+private _temperature = scriptNull; //Need the variable in this scope. So we need to init it here.
```

Comment:
I get it now. Can you just use:
``sqf
private "_temperature";
```
or
``sqf
private _temperature = nil;
```
Instead? This is really ugly.

Reply:
I was told `_temperature = nil;` works. But I wasn't sure. And didn't have time to test

Reply:
If it works then sure.
private "_temperature" would be as bad as private ARRAY

Reply:
Just tested. It works. Atleast in debug console.. Might break in scheduled but handleFired is unscheduled anyway